requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup
 lockfile
 jsonpointer
+requests
```

Comment:
@cdrage I'm confused. is python-requests not an rpm? It should be brought in just fine with the new way we do request processing in the dockerfiles.

also.. we should evaluate new dependencies pretty heavily from this point on. We should not do anything to break GA and adding a new dep may cause issues with release engineering. @kadel can you verify python-requests exists in CentOS/RHEL/Fedora ?